system/modules/calendar/languages/en/default.xlf

```diff
@@ -11,6 +11,9 @@
       <trans-unit id="MSC.cal_next">
         <source>&amp;gt;</source>
       </trans-unit>
+      <trans-unit id="MSC.cal_timeSeparator">
+        <source>–</source>
```

Comment:
The separator previously contained a whitespace before and after the dash, is that intentionally no longer present?